.travis.yml

```diff
@@ -1,12 +1,16 @@
+sudo: required
+dist: precise
+
 language: php
 php:
   - 5.5
-  - 5.6
+  - 7
 
-services:
-  - elasticsearch
+before_install:
+  - curl -O https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/2.2.0/elasticsearch-2.2.0.deb && sudo dpkg -i --force-confnew elasticsearch-2.2.0.deb
```

Comment:
@cjsaylor What do you think in make a matrix to test 1.7, 2.0, 2.1 and 2.2 since this is what the README says on the requirements?